security_drone_user_app/lib/presentation/image_banner.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+class ImageBanner extends StatelessWidget {
+  final String _path;
+
+  ImageBanner(this._path);
+
+  @override
+  Widget build(BuildContext context) {
+    return  Row(
+        mainAxisAlignment: MainAxisAlignment.start,
+      children: [Container(
+      constraints: BoxConstraints.expand(
+        height: 100.0,
```

Comment:
Make height and width as parameters of the constructor of the class, you can default them to 100.

Reply:
why?